wordpress-mobile/WordPress-iOS
# Menus: model service and tests

Adds the Menus model service and unit tests. The Menus service wraps around `MenuServiceRemote` with the CoreData model for persistence and relationships.

- `Menu` is a site menu itself.
- `MenuLocation` is a location/area in which a menu can appear on a site/theme.
- `MenuItem` are the individual items/link for a menu.

Currently, every sync of Menus is intended to wipe the `menus` and `menuLocations` relationships on `Blog`, not trying and update existing objects.

**To test:**
Run the tests under MenusServiceTests and check for all passing.

Needs review: @diegoreymendez would you please review? 🙈
